### ISSUE TITLE
Avoid usage of center_on_hmd() (e.g. in Teleport)

### ISSUE THREAD

**Issue by Windfisch** [GodotVR/godot-vr-common#4]
**tl;dr:** After `center_on_hmd()` has been called, it is very difficult to recover the absolute position in the physical space. Avoid calling this in the Teleport function so we don't lose the position in the user's physical space. Move/rotate the ARVROrigin instead.

**Long description**
When doing room-scale VR games, e.g. with the HTC Vive, it is important to maneuver the player back into the center of their physical space again, to maximize free walking into every direction. For example, the "Superhot VR" game does this by requiring the player to touch an object at the begin of every level.

Also, for the purpose of showing the user the room boundaries (->Chaperone system), knowledge of the physical position is necessary.

For OpenVR (and probably all other such systems), the ARVR's Camera position relative to the ARVROrigin equals the physical room positon, unless `center_on_hmd()` is called. Then, the said relative position has no physical equivalent any more.

_I'll do that once I've time_
